scripts/py_matter_idl/matter/idl/lint/lint_rules_parser.py

```diff
@@ -354,42 +358,6 @@ def CreateParser(file_name: str):
     """
     Generates a parser that will process a ".matter" file into a IDL
     """
-    return Parser(
-        Lark.open('lint_rules_grammar.lark', rel_to=__file__, parser='lalr', propagate_positions=True, maybe_placeholders=True), file_name=file_name)
-
-
-if __name__ == '__main__':
-    # This Parser is generally not intended to be run as a stand-alone binary.
-    # The ability to run is for debug and to print out the parsed AST.
-    import click
-
-    # Supported log levels, mapping string values required for argument
-    # parsing into logging constants
-    __LOG_LEVELS__ = {
-        'debug': logging.DEBUG,
-        'info': logging.INFO,
-        'warn': logging.WARN,
-        'fatal': logging.FATAL,
-    }
-
-    @click.command()
-    @click.option(
-        '--log-level',
-        default='INFO',
```

Comment:
Shouldn't this be 'info' not 'INFO' ?

Reply:
The option is `case_sensitive=False` so the case doesn't matter.